pages/common/export.md

```diff
@@ -0,0 +1,15 @@
+# export
+
+> Command to set environment variables.
+
+- Set a new environment variable:
+
+`export {{VARIABLE}}={{value}}`
+
+- Echo the value of an environment variable (don't forget the Dollar sign):
+
+`echo {{$VARIABLE}}`
```

Comment:
This example doesn't appear to be for the `export` command?

Reply:
You're right. Shall I remove that example?

Reply:
Yes.